lib/jsdom/living/nodes/HTMLImageElement-impl.js

```diff
@@ -29,10 +64,10 @@ class HTMLImageElementImpl extends HTMLElementImpl {
   }
 
   get height() {
-    if (this.hasAttribute("height")) {
-      return conversions["unsigned long"](this.getAttribute("height"));
-    }
-    return 0;
+    // Just like on browsers, if no width / height is defined, we fall back on the
+    // dimensions of the internal image data.
+    return this.hasAttribute("height") ?
```

Comment:
I was wondering about that. What if these limits were written as 0x7FFFFFFF? It'd become a bit more obvious why they are what they are.